autoflake.py

```diff
@@ -851,7 +851,7 @@ def fix_file(filename, args, standard_out):
                 filename)
             standard_out.write(''.join(diff))
     else:
-        if args.check:
+        if args.check and not args.quiet:
```

Comment:
This line is going to evaluate to `False` if the user does not pass `--quiet`, which is going to result in the `else` statement printing "Clean foo.py: nothing to fix". Instead, consider:
```
         if args.check:                                 
-            standard_out.write('No issues detected!\n')                                                                                                      
+            if not args.quiet:                                                                                                                               
+                standard_out.write('No issues detected!\n')                                                                                                  
         else:                                                                
             _LOGGER.debug('Clean %s: nothing to fix', filename)
```